src/layers/TralisLayer/TralisLayer.js

```diff
@@ -0,0 +1,14 @@
+import { TralisLayer as MBTTralisLayer } from 'mobility-toolbox-js/ol';
+
+class TralisLayer extends MBTTralisLayer {
+  constructor(options) {
+    // TODO this url should be set like others url
+    super({
+      url: 'wss://tralis-tracker-api.geops.io/ws',
```

Comment:
TODO

Reply:
Yes but this TODO intiates more changes in mobility-toolbox-js Websocket stuff (auto reconnection). Since it is already like that on prod I propose to do it in another PR as planned.